oauth.md

```diff
@@ -127,11 +127,13 @@ Key           | Value
 client_id     | Will be provided by Allthings, `xxxxxx` in the example below
 response_type | `code`
 redirect_uri  | The MicroApp URL, e.g. `https://www.example.com`
+scope         | Specifies the scope of the access (currently only `user:profile`)
+state         | Value used by the client to maintain state between the request and callback (for example an urlencoded json object)
 
 **Example URL:**
 
 ```
-https://api-sandbox.allthings.me/auth/authorize?client_id=xxxxxx&response_type=code&redirect_uri=https%3A%2F%2Fwww.example.com
+https://api-sandbox.allthings.me/oauth/authorize?client_id=xxxxxx&response_type=code&redirect_uri=https%3A%2F%2Fwww.example.com&state=%7b%22color%22%3a+%22green%22%2c+%22theme%22%3a+%22dark%22%7d&scope=user:profile
```

Comment:
Nitpick: The URL above listed the scope first (more important), the state second.

Reply:
More importantly, where in the authorisation code grant do I retrieve my state variable back?